src/app/issues-viewer/card-view/card-view.component.ts

```diff
@@ -50,4 +52,12 @@ export class CardViewComponent implements OnInit, AfterViewInit, OnDestroy, Filt
   retrieveFilterable(): FilterableSource {
     return this.issues;
   }
+
+  viewUserInBrowser() {
```

Comment:
Can we rename this to `switchToUserDetailPhase` or something similar?

I was confused by this method initially. I thought this method will open the user's github page in a new tab or browser window.